app/Modules/Recruitment/Models/JobAdvert.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Modules\Recruitment\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class JobAdvert extends Model
+{
+    protected $table = 'job_advert';
+    protected $guarded = ['id'];
+
+    public $timestamps = false;
+
+        protected $fillable = [
```

Comment:
Use guarded instead of fillable, so we don't need to adjust this every time a field is added in the database.